alyssa_mckee/function_assignments/oddeven.py

```diff
@@ -0,0 +1,26 @@
+def odd_even():
+	states = ["even", "odd"]
+	for num in range(1,2001):
+		print("Number is {}. This is an {} number.".format(num,states[num%2]))
```

Comment:
Very clever way of getting around having to use an if/else block to accomplish this same goal. Way to think outside the box!